test/gurka/gurka.h

```diff
@@ -594,6 +605,57 @@ findEdge(valhalla::baldr::GraphReader& reader,
   return std::make_tuple(baldr::GraphId{}, nullptr, baldr::GraphId{}, nullptr);
 }
 
+/**
+ * Finds an edge in the graph based on its begin and end node names
+ *
+ * @param reader           graph reader to look up tiles and edges
+ * @param begin_node_name  name of the begin node
+ * @param end_node_name    name of the end node
+ * @return the edge_id and its edge
+ */
+std::tuple<const baldr::GraphId, const baldr::DirectedEdge*>
+findEdgeByNodes(valhalla::baldr::GraphReader& reader,
```

Comment:
maybe we can delete the other method that does this, but i find it convenient to just say the begin and end node of the edge i want and just get that edge. this was very useful for the unit tests and is a less verbose and more intuitive way of specifying an edge via its OSM node names